dev-docs/bidder-adaptor.md

```diff
@@ -420,22 +420,50 @@ else if (rtbBid.rtb.native) {
 
     /* If yes, let's populate our response with native assets */
 
-    const native = rtbBid.rtb.native;
+    const nativeResponse = rtbBid.rtb.native;
 
     bid.native = {
-        title: native.title,
-        body: native.desc,
-        cta: native.ctatext,
-        sponsoredBy: native.sponsored,
-        image: native.main_img && native.main_img.url,
-        icon: native.icon && native.icon.url,
-        clickUrl: native.link.url,
-        impressionTrackers: native.impression_trackers,
+        title: nativeResponse.title,
+        body: nativeResponse.desc,
+        cta: nativeResponse.ctatext,
+        sponsoredBy: nativeResponse.sponsored,
+        image: nativeResponse.main_img && nativeResponse.main_img.url,
+        icon: nativeResponse.icon && nativeResponse.icon.url,
+        clickUrl: nativeResponse.link.url,
+        impressionTrackers: nativeResponse.impression_trackers,
     };
 }
 
 {% endhighlight %}
 
+As of the [0.34.1 release](https://github.com/prebid/Prebid.js/releases/tag/0.34.1), a bidder may optionally return the height and width of a native image asset.
```

Comment:
Height and width may be returned for `image` and `icon` assets

Reply:
Thanks @matthewlane - updated to note that info!